src/S3/S3Client.php

```diff
@@ -358,6 +358,20 @@ public function createPresignedRequest(CommandInterface $command, $expires, arra
         );
     }
 
+    /**
+     * Returns the URL to an object identified by its bucket and key.
+     *
+     * The URL returned by this method is not signed nor does it ensure the the
+     * bucket and key given to the method exist. If you need a signed URL, then
+     * use the {@see \Aws\S3\S3Client::createPresignedRequest} method and get
+     * the URI of the signed request.
+     *
+     * @param string $bucket  The name of the bucket where the object is located
+     * @param string $key     The key of the object
+     * @param array  $options Aws\Signature\SignatureInterface::presign options
```

Comment:
$options was added to the doc block in Aws\S3\S3ClientInterface by mistake.

Instead of adding it here, would you remove it there?  That should fix the deprecation notice.

Reply:
$options removed on both S3ClientInterface and S3Client itself